xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/ConfigurableClass.xml

```diff
@@ -82,8 +82,10 @@ $xwiki.jsfx.use('js/xwiki/actionbuttons/actionButtons.js', true)
       &amp;&amp; $currentDoc != 'XWiki.XWikiPreferences')
   ##
     #set($globaladmin = false)
+    #set($visibility = 'space')
   #else
     #set($globaladmin = true)
+    #set($visibility = 'wiki')
```

Comment:
Actually there's no need to set any variable since the $editor one already represents that info, no?